doc/data/messages/u/unneeded-not/good.py

```diff
@@ -1 +1,2 @@
-# This is a placeholder for correct code for this message.
+if not input():
```

Comment:
```suggestion
if input():
```

Reply:
I thought the good example intend to check for `not input()` but the bad example is mistakenly put duplicated `not'.

Reply:
👍 I think in this case doing the same thing in both example is possible and match the message name more.

Reply:
👍 Agree with you.